pkg/apiserver/webhooks/datavolume-mutate_test.go

```diff
@@ -0,0 +1,186 @@
+/*
+ * This file is part of the CDI project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2019 Red Hat, Inc.
+ *
+ */
+
+package webhooks
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"encoding/json"
+	"fmt"
+
+	"github.com/appscode/jsonpatch"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/api/admission/v1beta1"
+	authorization "k8s.io/api/authorization/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	fakeclient "k8s.io/client-go/kubernetes/fake"
+	k8stesting "k8s.io/client-go/testing"
+
+	cdicorev1alpha1 "kubevirt.io/containerized-data-importer/pkg/apis/core/v1alpha1"
+	"kubevirt.io/containerized-data-importer/pkg/controller"
+)
+
+var _ = Describe("Mutating DataVolume Webhook", func() {
+	Context("with DataVolume admission review", func() {
+		key, _ := rsa.GenerateKey(rand.Reader, 2048)
+
+		It("should reject review without request", func() {
+			ar := &v1beta1.AdmissionReview{}
+
+			resp := mutateDVs(key, ar, true)
+			Expect(resp.Allowed).To(BeFalse())
+			Expect(resp.Result.Message).ShouldNot(BeEmpty())
```

Comment:
Do we care about what the message is? And do we know what it should be if it is rejected?